mmdet3d/models/backbones/mink_resnet.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, depth, in_channels, num_stages=4, pool=True):
 
         for i, num_blocks in enumerate(stage_blocks):
             setattr(
-                self, f'layer{i}',
+                self, f'layer{i + 1}',
```

Comment:
why add 1 for layer?

Reply:
The original implement in the master branch is `layer{i+1}`, which starts with 1 and we changed it in the refactoring process.  If we don't do this, the checkpoint in the README will not be loaded correctly.